tests/data/norovirus.mutations.txt

```diff
@@ -0,0 +1,137 @@
+NC_001959.2	422	A	G	3
```

Comment:
Is this file important test data? It does not matter for the classification, does it?

Reply:
It is not, will remove it.